sklearn/model_selection/tests/test_split.py

```diff
@@ -48,6 +48,29 @@
 
 from sklearn.svm import SVC
 
+SIMPLE_SPLITTERS = [
```

Comment:
"simple" seems a little too generic. I think this means "simple" means "no groups"? So maybe:

```suggestion
NO_GROUP_SPLITTERS = [
```